src/seedu/addressbook/parser/Parser.java

```diff
@@ -307,7 +307,7 @@ private static boolean isPrivatePrefixPresent(String matchedPrefix) {
         return new HashSet<>(tagStrings);
     }
     /**
-     * Parses arguments in the context of the EditFeesCommand command.
+     * Parses arguments in the context of the edit fees command
      *
```

Comment:
Lawl renaming all these makes less sense. We are talking from a technical perspective

Reply:
Hahaha I know, but I thought I would make it consistent. Took only a minute to do anyway

Reply:
I think the Pascal case makes more sense here as we are referring to classes, not random nouns we use everyday

Reply:
Oh, I think the original commands in the parser had it like this so I followed the same. I can change it though, no prob. Should I?

Reply:
I think you should. Feels weird to use those as proper nouns

Reply:
Ok!